styles/about.css

```diff
@@ -0,0 +1,33 @@
+body {
```

Comment:
I like the separation of CSS into different files! Definitely helps with adding custom CSS for specific pages.